### DISCUSSION TITLE
Improve ability to switch between multiple list views in admin UI

### DISCUSSION THREAD

**Discussion by swheeler7** [payloadcms/payload#4465] [Feature Requests & Ideas]
I really like the tabs feature that was introduced in 2.0 for edit views, but as far as I can tell list views don't have the same feature (you can currently replace the default but there is no interface to switch between multiple views). I would love to see something similar for list views to enable tab switching between the default list view and custom list views in the admin interface. 

This would make it easier for plugin authors to create one or more list views that complement the default list view but don't replace it. It would also make it easier for list view plugins to work alongside each other without implementing their own method of switching between list views - see this plugin for example [https://github.com/DennisSnijder/payload-workflow](https://github.com/DennisSnijder/payload-workflow)

**Comment by 58bits** [comment]
Hey @swheeler7 - this would be great. For interest, and indirectly related - we've implemented a custom view and will also implement our own 'switch' to toggle between views, but we ran into this issue - https://github.com/payloadcms/payload/issues/4990
